### DISCUSSION TITLE
install/uninstall

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2311] [Announcements]
# install/uninstall

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/uninstall.html

**Comment by Ye-Ting** [comment]
删除 NS 的执行结果, 发现还有 kcollie-system ns 存在

**Reply by windsonsea** [reply]
Fixed in PR #5290 ，目前社区版中已经移除 kcollie 这个组件了

**Comment by KubeKyrie** [comment]
能否可以补充商业版的卸载步骤

**Reply by windsonsea** [reply]
商业版通常需要铲掉整个环境
